xbmc/addons/AddonDatabase.cpp

```diff
@@ -218,7 +218,7 @@ void CAddonDatabase::SyncInstalled(const std::set<std::string>& ids, const std::
     BeginTransaction();
     for (const auto& id : added)
       m_pDS->exec(PrepareSQL("INSERT INTO installed(addonID, enabled, installDate) "
-          "VALUES('%s', '%d', '%s')", id.c_str(), enabled.find(id) != enabled.end() ? 1 : 0, now.c_str()));
+          "VALUES('%s', 1, '%s')", id.c_str(), now.c_str()));
```

Comment:
The `enabled` parameter added in 108db2206194651cc72883b344f9911707a386c3 looks to be unused now